src/firecfg/firecfg.config

```diff
@@ -23,6 +23,7 @@ Maps
 Mathematica
 Natron
 PCSX2
+Postman
 PPSSPPQt
 PPSSPPSDL
```

Comment:
```suggestion
PPSSPPQt
PPSSPPSDL
Postman
```

This file appears to use C/ASCII collation, so sort using that.

(Sorry for the back-and-forth)

Reply:
> This file appears to use C/ASCII collation, so sort using that.

Interesting. If there's a way to use an easier sorting mechanism we should look into that IMO.